clients/naming_client/push_receiver.go

```diff
@@ -91,9 +91,12 @@ func (us *PushReceiver) startServer() {
 	}
 
 	defer conn.Close()
```

Comment:
conn.Close should be inside goroutine, or the connection will Close when startServer completes